### DISCUSSION TITLE
start myMPD on different start site

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#323] [Q&A] [answered]
Is it possible to start myMPD on a different start page?
- what is working: start up myMPD with 
**http://IP:Port/**
- what is not working: start up myMPD with
**http://IP:Port/#/Browse/Database/Album!0/-/-/** or **http://IP:Port/#/Browse/Covergrid!0/AlbumArtist/AlbumArtist/**
 -> starting ending always in a preselected Album or covergrid with pre-filled search field like following enty:
**Album: ?&TS318202021303781** or **?&TS3182020213336944**

Is it possible to select the start site on an other way than the full path to album or covergrid?

**Comment by jcorporation** [comment, marked as answer]
myMPD interpretes the hash as uri to open. For example this works for me: https://jukebox/#/Browse/Database/AlbumArtist!0/-/-/Bauhaus 

I am not sure if I understand your question correctly...

**Comment by payback007** [comment]
My Problem is, the URL-call is: http://IP:Port/#/Browse/Database/Album!0/-/-/ but myMPD is opening with this URI: http://IP:Port/#/Browse/Database/Album!0/-/-/?&TS318202021303781. Extension seem to be done automatically.

**Comment by jcorporation** [comment]
`?&TS318202021303781` this is nothing from myMPD it must be from your environment.

**Comment by payback007** [comment]
update: problem solved -> after activation of "caching capability" the problem is gone. I could also reproduce: 
- caching off: **?&TS318202021303781** was default search value
- caching on: starting of request site with http://IP:Port/#/Browse/Database/Album!0/-/-/ as expected

thanks for the support!
